packages/agent/src/agents/core/snakAgent.ts

```
@@ -612,26 +722,15 @@
-
-            if (
-              (chunk.name === 'Branch<tools,tools,agent,end>' ||
-                chunk.name === 'Branch<agent,tools,agent,human,end>') &&
-              chunk.event === 'on_chain_end'
-            ) {
-              lastChunkToSave = chunk;
+            lastChunk = chunk;
+            const state = await app.getState(executionConfig);
+            let isNewNode: boolean = false;
+            if (currentAgent && currentAgent != state.last_agent) {
+              isNewNode = true;
```

ESLint

[error] 'isNewNode' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)